src/TransCreator/DateSelector.jsx

```diff
@@ -1,37 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { MaterialIcons } from '@expo/vector-icons';
-import {
-  Text,
-  DatePicker,
-} from 'native-base';
-import { View } from 'react-native';
+import { View, Text } from 'react-native';
+import DateTimePicker from 'react-native-modal-datetime-picker';
 import moment from 'moment';
 import PropTypes from 'prop-types';
 import styles from './Style';
 
-
-const DateSelector = ({ transDate, setTransDate }) => (
-  <>
-    <MaterialIcons name="date-range" style={styles.dateItemIcon} />
-    <Text>Date: </Text>
-    <View style={styles.dateView}>
-      <DatePicker
-        defaultDate={new Date()}
-        minimumDate={new Date(1990, 1, 1)}
-        maximumDate={new Date(2050, 12, 31)}
-        locale="en"
-        modalTransparent={false}
-        animationType="fade"
-        androidMode="default"
-        placeHolderText={moment.unix(transDate).format('MM/DD/YYYY')}
-        textStyle={{ color: '#5C6BC0' }}
-        placeHolderTextStyle={{ color: '#5C6BC0' }}
-        onDateChange={(newDate) => setTransDate(moment(newDate).unix())}
-        disabled={false}
-      />
-    </View>
-  </>
-);
+const DateSelector = ({ transDate, setTransDate }) => {
+  const [datePickerShow, setDatePickerShow] = useState(false);
+  return (
+    <>
+      <MaterialIcons name="date-range" style={styles.dateItemIcon} />
+      <Text>Date: </Text>
+      <View style={styles.dateView}>
+        <Text
```

Comment:
cannot we use `native-base`'s `Text` component

Reply:
We are not goIng to use native-base any more, all the previous implementations should get rid of native-base components little by little. Thus, I used Text from 'react-native' here.

Reply:
@MichaelCTH then please create a story for this in jira